core/config/v2/docs/core.toml

```diff
@@ -166,6 +166,8 @@ AllowOrigins = 'http://localhost:3000,http://localhost:6688' # Default
 # Usually this will be the same as the URL/IP and port you use to connect to the Chainlink UI.
 BridgeResponseURL = 'https://my-chainlink-node.example.com:6688' # Example
 # **ADVANCED**
+# BridgeCacheTTL controls the cache TTL for all bridge tasks to use old values in newer observations in case of intermittent failure.
+BridgeCacheTTL = '0s' # Default
```

Comment:
Oh the `// **ADVANCED**` line goes below with `HTTPWriteTimeout`

Reply:
And we don't have a strict rule, but maybe consider putting this before `BridgeResponseURL` to stay alphabetical?
...I see that HTTP* breaks the rule right after this though :facepalm: